devel/ansible/roles/hotness-dev/files/fedmsg.d/hotness.py

```diff
@@ -48,8 +47,14 @@
     'hotness.koji': {
         'server': 'https://koji.fedoraproject.org/kojihub',
         'weburl': 'http://koji.fedoraproject.org/koji',
-        'cert': os.path.expanduser('~/.fedora.cert'),
-        'ca_cert': os.path.expanduser('~/.fedora-server-ca.cert'),
+        # Kerberos configuration to authenticate with Koji. In development
+        # environments, use `kinit <fas-name>@FEDORAPROJECT.ORG` to get a
+        # Kerberos ticket and set all krb_* settings to `None`.
+        'krb_principal': None,
+        'krb_keytab': None,
+        'krb_ccache': None,
+        'krb_proxyuser': None,
```

Comment:
I really just put it in there to expose the Koji auth API as-is so people can do whatever they want. I don't know that there are any other deployments of the-new-hotness out there besides ours so maybe this is a moot point, but I erred on the side of being flexible, even though we don't need that flexibility.